Update pricing features and add Contact Sales dialog

bouncei opened the pull request:
- Added "1 month FREE TRIAL" feature to all pricing plans for clarity.
- Updated FAQ questions to emphasize "FREE TRIAL" terminology.
- Introduced Contact Sales dialog for Enterprise plan and FAQs to facilitate user inquiries.

bouncei committed:
Update pricing features and add Contact Sales dialog

- Added "1 month FREE TRIAL" feature to all pricing plans for clarity.
- Updated FAQ questions to emphasize "FREE TRIAL" terminology.
- Introduced Contact Sales dialog for Enterprise plan and FAQs to facilitate user inquiries.
- Added .vercel to .gitignore for deployment configuration.

devfunsho commented on src/components/ContactSalesForm.tsx:
```diff
@@ -0,0 +1,534 @@
+  "mailinator.com",
+  "temp-mail.org",
+  "throwaway.email",
+];
```
You can move this to a constant file. Maybe a `common.ts` file for generic constants

bouncei replied:
Sure. Will do

devfunsho commented on src/components/ContactSalesForm.tsx:
```diff
@@ -0,0 +1,534 @@
+  "New Zealand",
+  "South Korea",
+  "Other",
+];
```
Any particular reason you are limiting to these countries only? Also, can you switch to a package that list out countries or move this to a constant file

bouncei replied:
Yeah. Not done yet. I was using this to test the ingestion endpoint

bouncei committed:
Refactor ContactSalesForm component to use constants for schema and options

bouncei committed:
Update dependencies and enhance ContactSalesForm with phone number input

- Added `react-phone-number-input` and `react-select-country-list` for improved user experience in the Contact Sales form.
- Updated phone number validation to ensure valid input using `isValidPhoneNumber`.
- Refactored country selection to utilize `react-select-country-list` for dynamic country options.
- Added phone input styles for better UI presentation.
